pursuit.cabal

```diff
@@ -85,7 +85,7 @@ library
                  , directory                     >= 1.1        && < 1.3
                  , warp                          >= 3.0        && < 4
                  , data-default
-                 , aeson                         >= 0.6        && < 0.10
+                 , aeson                         >= 0.6        && < 0.12
```

Comment:
Since we are using aeson 0.11.x, I'd prefer to bump up the lower bound to 0.11, so that we can't break the format just by choosing a different version of aeson.